src/components/ContinuationIn/UploadAffidavit.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-expand-transition>
+    <!-- Upload Affidavit -->
+    <v-row
+      v-if="isContinuationInAffidavitRequired"
+      class="mt-6"
+      no-gutters
+    >
+      <v-col
+        cols="12"
+        sm="3"
+      >
+        <label>Upload Affidavit</label>
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="9"
+      >
+        <p>
+          Upload the affidavit from the directors.
+        </p>
+
+        <ul>
+          <li>Use a white background and a legible font with contrasting font colour</li>
+          <li>PDF file type (maximum 30 MB file size)</li>
+        </ul>
+
+        <FileUploadPreview
+          class="mt-4"
+          inputFileLabel="Affidavit from directors"
+          :maxSize="MAX_FILE_SIZE"
+          :pdfPageSize="PdfPageSize.LETTER_SIZE"
+          :hint="business.affidavitFile ? 'File uploaded' : undefined"
+          :inputFile="business.affidavitFile"
+          :showErrors="getShowErrors"
+          :customErrorMessage.sync="customErrorMessage"
+          :isRequired="getShowErrors"
+          @fileValidity="onFileValidity($event)"
+          @fileSelected="onFileSelected($event)"
+        />
+      </v-col>
+    </v-row>
+  </v-expand-transition>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { StatusCodes } from 'http-status-codes'
+import { DateMixin, DocumentMixin } from '@/mixins'
+import { ExistingBusinessInfoIF, PresignedUrlIF } from '@/interfaces'
+import { PdfPageSize } from '@/enums'
+import FileUploadPreview from '@/components/common/FileUploadPreview.vue'
+@Component({
+  components: {
+    FileUploadPreview
+  }
+})
+export default class UploadAffidavit extends Mixins(DateMixin, DocumentMixin) {
+  readonly PdfPageSize = PdfPageSize
+
+  @Getter(useStore) getExistingBusinessInfo!: ExistingBusinessInfoIF
+  @Getter(useStore) getKeycloakGuid!: string
+  @Getter(useStore) getShowErrors!: boolean
+  @Getter(useStore) isContinuationInAffidavitRequired!: boolean
```

Comment:
You won't need this when you move the v-row back into the parent component.

Reply:
Thanks, updated. Moved the v-row back into the parent component.